container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -2102,6 +2199,57 @@ configuration:
     default: support@example.com
     description: Support contact information for the cluster
     required: true
+  - name: SYSLOG_ADAPT_CERT
+    secret: true
+    generator:
+      id: syslog_adapt_cert
+      type: Certificate
+      value_type: certificate
+    description: PEM-encoded certificate
+    required: true
+  - name: SYSLOG_ADAPT_KEY
+    secret: true
+    generator:
+      id: syslog_adapt_cert
+      type: Certificate
+      value_type: private_key
+      key_length: 4096
```

Comment:
The `key_length` isn't used by the secrets generator and can/should be omitted.

Reply:
Will do.

Reply:
See commit 84c47e4c94470197738340ded7aadd7453b71f56